src/main/java/uk/gov/digital/ho/hocs/queue/ukvi/UKVIComplaintConsumer.java

```diff
@@ -61,13 +61,7 @@ public void configure() {
 
     @PreDestroy
     public void onExit() {
-        try {
-            log.info("Sleeping {} seconds to allow in-flight threads to complete.", shutdownWaitSeconds);
-            Thread.sleep(shutdownWaitSeconds * 1000);
-        } catch (InterruptedException e) {
-            log.error("Sleep failed : {}", e.getMessage());
-        }
-        log.info("Sleep done");
+        log.info("hocs-case-creator stopping gracefully");
```

Comment:
Medium: You don't need the `hocs-case-creator` part as this will be added into the log in the correct place.

Reply:
Good point, I did it like that everywhere else, so for consistancy could stay or change everywhere?
Its not the worst thing to log when filtering in kibana :)

Reply:
You should remove it, as the application name is set by a property which is used elsewhere..

Reply:
@gardnersam Please change to "Stopping gracefully". I'll change the other projects as I work on them.

Reply:
Updated logging message